tests/detect-ldap-result/ldap.syn

```diff
@@ -0,0 +1,2 @@
+flow default tcp 1.1.1.1:5555 > 2.2.2.2:389 (tcp.initialize; mss:9000;);
+default < (content:"\x30\x1f\x02\x01\x02\x65\x1a\x0a\x01\x04\x04\x00\x04\x13\x53\x69\x7a\x65\x20\x6c\x69\x6d\x69\x74\x20\x65\x78\x63\x65\x65\x64\x65\x64";);
```

Comment:
If this is synthetic, I would put another message than `"Size limit exceeded"` to avoid confusion, like `"Message size limit got exceeded : max 65k"`